doc/Tutorials.xml

```diff
@@ -181,12 +181,108 @@ There are various examples of packages using &AutoDoc; for only
 <P/>
 
-<!--
-TODO: Explain how to mix AutoDoc content with existing &GAPDoc; XML files.
--->
+<Subsection Label="Tut:IntegrateExisting:EverythingThere">
+  <Heading>Using &AutoDoc; on a complete &GAPDoc; manual</Heading>
+
+Suppose you already have a complete XML manual, with some main and title
+XML files and some documentation for operations distributed over
+all your <F>.g{,d,i}</F> files. Even then &AutoDoc; can do something for you.
+Suppose your main xml file is named <F>PACKAGENAME.xml</F> and is in the
+<F>/doc</F> subfolder of your package. Then calling
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false );
+</Listing>
+will rebuild your manual. The main point in favor of doing is that you do
+not have to keep a list of all the <F>.g{,d,i}</F> files you have manual entries in.
+<P/>
+But there is more. &AutoDoc; can create a <F>maketest.g</F> file, which uses the
+examples in your manual to test your package. This can be achieved via
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false, maketest := true );
+</Listing>
+Now the file <F>maketest.g</F> appears in your package directory, and
+<Listing>
+gap maketest.g
+</Listing>
+test the examples from your manual.
+</Subsection>
 
-</Section>
+<Subsection Label="Tut:IntegrateExisting:GapDocOptions">
+<Heading>Setting different &GAPDoc; options</Heading>
 
+Sometimes, the default values for the &GAPDoc; command used by &AutoDoc;
+may not be suitable for your manual.
+<P/>
+Suppose you have your main xml file called <F>main.xml</F>. Then you can tell &AutoDoc;
+to use this file as the main xml file via
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false,
+           gapdoc := rec( main := "doc/main.xml" ) );
+</Listing>
+<P/>
+Also, it might happen that your <F>g.{,d,i}</F> files are not in somehow standard named
+subdirectories of your package, but maybe in a folder called <F>package_sources</F>.
+Then you can tell &AutoDoc; to search this folder instead for the standard ones with
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false,
+           gapdoc := rec( scan_dirs := [ "package_source" ] ) );
+</Listing>
+It is also possible to give specific files, maybe hidden somewhere really deep in your
+packages folder structure.
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false,
+           gapdoc := rec( files := [ "path/to/some/hidden/file.gds" ] ) );
+</Listing>
+Giving such a file does not prevent the standard <C>scan_dirs</C> from being scanned for
+other files.
+<P/>
+Next, &GAPDoc; supports the documentation to be build with relative paths.
+This means, links to documentations of other packages or the &GAP; library will
+not be absolute, but relative from your documentation. This can be particulary useful
+if you want to build a release tarball or move your &GAP; installation around later.
+Suppose you are building your documentation in the <F>doc</F> subfolder of your package
```

Comment:
I find this sentence confusing. It sounds as if you are talking about invoking `AutoDoc()` from within the `doc/` subdirectory, something that normally is never done. So I am not quite sure what you mean here...